sherpa/optmethods/__init__.py

```diff
@@ -372,8 +372,8 @@ class LevMar(OptMethod):
        free parameters.
     epsfcn : number
        This is used in determining a suitable step length for the
-       forward-difference approximation; default is sqrt( DBL_EPSILON
-       ) ~ 1.19209289551e-07, where DBL_EPSILON is the smallest number
+       forward-difference approximation; default is FLT_EPSILON ~ 1.19209289551e-07,
+       where DBL_EPSILON is the smallest number
```

Comment:
I have been working with a 120 char limit in mind (and in the tools config), but I guess you are saying that we want to keep the docstring at the 70-80ish level, right?